src/ui/App.tsx

```diff
@@ -133,10 +136,11 @@ const App = () => {
     <IonApp>
       <AppWrapper>
         <StrictMode>
-          {stateCache.initialized ? (
+          {initialized ? (
             <>
               {renderApp()}
               {!isPublicPage && !authentication.loggedIn ? <LockPage /> : null}
+              {!isOnline ? <AppOffline /> : null}
```

Comment:
I would've thought we would just do this in AppWrapper and show the login page even if we're offline.

Reply:
Sure. I setup `z-Index` of `AppOffline` less than `LockPage` but greater than other pages and modal. Therefore `LockPage` alway display on top of app.

https://github.com/user-attachments/assets/577ee0c2-7697-42fa-bfc9-da09f4e71d7c

Reply:
ok cool thanks :)